src/ray/core_worker/core_worker.cc

```diff
@@ -384,11 +384,13 @@ CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_
   NodeID local_raylet_id;
   int assigned_port;
   std::string serialized_job_config = options_.serialized_job_config;
+  // TODO(architkulkarni): Add runtime env to this, ray.worker.connect in default_worker,
```

Comment:
I'll remove this after CI finishes running, don't want to keep restarting it

Reply:
We can remove this now right?